tests/test_ec2/test_instances.py

```diff
@@ -50,6 +51,7 @@ def test_instance_launch_and_terminate():
     instances.should.have.length_of(1)
     instances[0].id.should.equal(instance.id)
     instances[0].state.should.equal('running')
+    datetime.datetime.strptime(instances[0].launch_time, '%Y-%m-%dT%H:%M:%S.%f')
```

Comment:
I've used freezegun to make a slightly stronger assertion here: https://github.com/spulec/moto/commit/b1c41820f70e8d9c8464f5fc380fb8a2fae1c1d5#diff-314909133cd9b2301075555c88d76574R56